app/screens/planner/planner-screen.tsx

```diff
@@ -110,6 +110,12 @@ export const PlannerScreen = observer(function PlannerScreen({ navigation }: Pla
     onDateChange(date)
   }
 
+  const handleNow = () => {
+    onDateChange(new Date())
+    setDatePickerVisibility(false)
```

Comment:
I think it's better to let the user close the picker by clicking on the confirmation button. 
Less surprising and maintains consistency with the rest of the tabs

Reply:
I think the main goal of the now button is to get to the relevant routes for now in a hurry, as fast as possible. I also don't like the UI of the date picker in Apple Maps because unlike the current solution which is very fast to navigate, the UI on Apple Maps has a lot of friction and tiny controls for the time (which is way more important than the date IMO).
Perhaps a better solution is the outside modal selection mocked up here: https://github.com/guytepper/better-rail/issues/60#issuecomment-933718049